modules/run-consul/run-consul

```diff
@@ -299,23 +299,33 @@ EOF
   fi
 
   local rpc_encryption_configuration=""
-  if [[ "$enable_rpc_encryption" == "true" && ! -z "$ca_path" && ! -z "$cert_file_path" && ! -z "$key_file_path" ]]; then
+  if [[ "$enable_rpc_encryption" == "true" && ! -z "$ca_path" ]]; then
     log_info "Creating RPC encryption configuration"
     rpc_encryption_configuration=$(cat <<EOF
 "verify_outgoing": true,
-"verify_incoming": true,
 "verify_incoming_https": false,
```

Comment:
If you're removing the `verify_incoming` here it would probably make sense to also remove the verify_incoming_https as that's now just noise.